src/main/java/teammates/storage/entity/Entity.java

```diff
@@ -0,0 +1,20 @@
+package teammates.storage.entity;
+
+import java.lang.reflect.Field;
+
+import javax.jdo.annotations.PrimaryKey;
+
+final class Entity {
```

Comment:
Hmmm is it possible to make this class `abstract`and repurpose it as the superclass of all entities instead? Then the `getField...` method can be `protected static` instead, and you might actually not need the empty constructor. Likely PMD will complain something about abstract class without abstract methods, but design-wise it looks more sound as opposed to current design in which this class seems like nothing but the container of the `getField...` method.